careers/ladder.md

```diff
@@ -28,73 +28,161 @@ IC0 > IC1 > IC2 > IC3 > IC4 > IC5
+- Understands team mission, goals and backlog.
+
+#### Discretion
+- Guided by objectives of project or ongoing assignments.
+- Plans own work priorities on a regular basis.
+
 
 ### IC2 Engineer
 
+#### tl;dr: is an effective team member
+
+#### Knowledge Leadership
 - Consistently writes and delivers correct and clean quality code with guidance.
+- Owns a small-to-medium feature from technical design through completion.
+
+#### Impact
+- Makes steady, well-paced progress without the need for constant significant feedback from more senior engineers.
+
+#### Influence
 - Self-sufficient and makes steady progress on tasks.
 - Knows when to ask for help and how to get unblocked.
-- Makes steady, well-paced progress without the need for constant significant feedback from more senior engineers.
-- Owns a small-to-medium feature from technical design through completion.
+- Effectively partners with other engineers.
 - Provides help and support outside of area under their responsibility.
 - Participates in the interviewing process of new team members.
 
+#### Discretion
+- Actively engages in team rituals.
+- Understands business impact of their work.
+
 ### IC3 Senior Engineer
 
-- Owns a service or large technology component.
-- Requires minimal direction or oversight, delivers quickly.
-- Understands elements of the business that their work supports.
+#### tl;dr: can drive change and impact at the team level
+
+#### Knowledge Leadership
 - Understands and makes well-reasoned design decisions and tradeoffs in their area.
 - Works in other areas of the codebase with guidance from owners of that system.
+- Owns a service or large technology component.
+
+#### Impact
+- Delivers significant business value at team level.
+- Helps the team deliver on its overall goals.
+- Can drive specific initiatives.
+
+#### Influence
 - Persistent in the face of roadblocks, dispatches them efficiently, pulling in others as necessary.
-- Has end-to-end responsibility on projects of increasing complexity and contributes to code that is shared across
-  projects.
+- Has end-to-end responsibility on projects of increasing complexity and contributes to code that is shared across projects.
+- Effectively partners with team's cross-functional partners (including PM, Designer, Data Analyst).
 - Mentors junior engineers.
+
+#### Discretion
+- Requires minimal direction or oversight, delivers quickly.
+- Understands elements of the business that their work supports.
+- Demonstrates fluency with team KPIs and key drivers of these KPIs.
 - Demonstrates knowledge of industry trends, infrastructure and systems.
 
 ### IC4 Staff Engineer
 
+#### tl;dr: can drive change and impact across multiple teams
+
+#### Knowledge Leadership
 - Owns large scale and impact service or collection of services.
-- Understands the Artsy business and makes decisions based on its direction.
-- Sought out by other team members for technical guidance.
+
+#### Impact
```

Comment:
This might fall outside the scope of this PR, but I've always felt that our ladder misses the qualitative, hard-to-measure aspects of leaderships at IC4 (and IC5). This current ladder implies growing into IC4 requires only advancing one's technical skills, and excludes some skills that we do (in practice) value at senior levels: good communication, empathy, decisiveness. Maybe we could expand this slightly to include those?

Reply:
* I think good communication [should already be expected at IC3](https://github.com/artsy/README/pull/188/files#r281567943).

* Decisiveness seems already captured by:

   > Requires minimal direction or oversight

   and

   > Understands the Artsy business and makes decisions based on its direction

* Empathy is an interesting one. I think it should implicitly apply to all levels through “People Are Paramount” and “being nice is nice”, but I do think that at these levels we could be a bit more clear of some expectations. As you know, I’m a fan of [LARA](https://sparqtools.org/lara/), are you referring to things along those lines?

Reply:
For sure. I'll distill my point down to this: while good communication and empathy are captured by the current ladder, neither is referred to explicitly by name. Further, it's not clear how different levels of the ladder have different requirements around communication and empathy. 

The current ladder implies that, for example, once someone accomplishes good communication at IC3, those same skills are sufficient to advance through the rest of the ladder. In practice, that's not the case. So can we capture the differences in how Artsy expects IC2/IC3/IC4's to communicate? To empathize?